codes/classical/properties/Low Density Parity Check (LDPC)

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
ldpc.yml should be the file name. The paper doi is Kitaev's paper on quantum computation, so it's unclear which classical codes you're trying to discuss here. This entry should be about defining what it means for a code to be LDPC, and what types of codes can be LDPC. SPecific examples of LDPC codes should be their own yml files.

Reply:
Sorry, I put the cite incorrectly. I will change it. Can I do the changes here or should I re upload the corrected file?